project-1-flexbox/Style.css

```diff
@@ -0,0 +1,188 @@
+
+
+
+* {
+    box-sizing: border-box;
+}
+
+
+
+
+
+
+
+
+.container {
+	padding: 30px;
+	margin-right:20px
+
+	
+}
+
+
+
+.header {
+	display: flex;
+	flex-direction: row; 
+	
+
+}
+nav {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+}
```

Comment:
It would have been nice to see the links with a the font colour in the given image.